### PR TITLE
use struct init shorthand in lib.rs

### PR DESCRIPTION
**Pull request by ericmarkmartin**
Implements # .

- [x] cargo fmt has been run
- [x] cargo test has been run and passes
- [ ] documentation has been updated with relevant examples (if relevant)

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -664,13 +660,11 @@ impl<T> Webauthn<T> {
     {
         let chal = self.generate_challenge();
 
-        let verified = creds.iter()
-            .all(|cred| cred.verified);
-        let unverified = creds.iter()
-            .all(|cred| !cred.verified);
+        let verified = creds.iter().all(|cred| cred.verified);
```
Comment (Firstyear) [inline src/lib.rs:663]: Did you manually change these or did rust fmt do it?

Comment (ericmarkmartin) [reply src/lib.rs:663]: This was `cargo fmt`

Comment (Firstyear) [reply src/lib.rs:663]: No problem then was curious is all. What version of rust are you using? I've seen some issues with cargo fmt in the past where different versions output different code and so if people have different versions they end up fighting over the formatting ....

Comment (ericmarkmartin) [reply src/lib.rs:663]: The active toolchain section from `rustup show` reads:
```
stable-x86_64-apple-darwin (default)
rustc 1.47.0 (18bf6b4f0 2020-10-07)
```

Comment (Firstyear) [reply src/lib.rs:663]: Cool no problems, that explains it. I'm still using 1.45.2.

**Comment by Firstyear** [comment]
Thanks, looks good!

**Pull request merged**